posthog/models/cohort/test/test_util.py

```diff
@@ -0,0 +1,360 @@
+from posthog.models.cohort import Cohort
+from posthog.models.cohort.util import get_dependent_cohorts, simplified_cohort_filter_properties
+from posthog.test.base import BaseTest, _create_person, flush_persons_and_events
+
+
+def _create_cohort(**kwargs):
+    team = kwargs.pop("team")
+    name = kwargs.pop("name")
+    groups = kwargs.pop("groups")
+    is_static = kwargs.pop("is_static", False)
+    cohort = Cohort.objects.create(team=team, name=name, groups=groups, is_static=is_static)
+    return cohort
+
+
+class TestCohortUtils(BaseTest):
```

Comment:
Main tests are below in a separate class.

These are some old tests that should've been committed but weren't. (They are very old, never realised it because it was a 'new' file vs addition to existing file, was never staged).